buildSrc/src/main/java/Dependencies.kt

```diff
@@ -0,0 +1,83 @@
+import org.codehaus.groovy.ast.Variable
+
+object Versions {
+    // App versioning
+    const val Demo_appId = "org.openmined.syft.demo"
+    const val appVersionCode = 1
+    const val appVersionName = "1.0"
+
+    const val compileSdk = 29
+    const val minSdk = 24
+    const val targetSdk = 29
+
+    const val gradle = "3.5.3"
+    const val kotlin = "1.3.61"
+    const val buildTools = "29.0.2"
+
+    // Android libraries
+    const val appCompat = "1.1.0"
+    const val coreKtx= "1.2.0"
+    const val kotlinSerialization = "0.14.0"
+
+    // Tools
+    const val rxJava = "2.2.7"
+    const val rxAndroid = "2.1.1"
+    const val webrtc = "1.0.30039"
+    const val okhttp = "4.3.1"
+    const val protobuf = "3.11.0"
+    const val kotlinSyft = "0.0.7"
+
+    //neflix
```

Comment:
rename here as well. Rather than saying netflix, release management is a better comment